clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/BaseIntegrationSpec.scala

```diff
@@ -0,0 +1,101 @@
+package org.broadinstitute.clio.integrationtest
+
+
+import org.broadinstitute.clio.client.webclient.ClioWebClient
+import org.broadinstitute.clio.server.dataaccess.elasticsearch.ElasticsearchIndex
+import org.broadinstitute.clio.util.json.ModelAutoDerivation
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.client.RequestBuilding
+import akka.http.scaladsl.model._
+import akka.stream.ActorMaterializer
+import akka.testkit.TestKit
+import com.sksamuel.elastic4s.http.HttpClient
+import com.sksamuel.elastic4s.http.index.mappings.IndexMappings
+import com.typesafe.scalalogging.LazyLogging
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import io.circe.Printer
+import org.apache.http.HttpHost
+import org.elasticsearch.client.RestClient
+import org.scalatest.{AsyncFlatSpecLike, BeforeAndAfterAll, Matchers}
+
+import scala.sys.process._
+
+import java.util.UUID
+
+/**
+  * Base class for Clio integration tests, agnostic to the location
+  * of the Clio / Elasticsearch instances that will be tested against.
+  */
+abstract class BaseIntegrationSpec(clioDescription: String)
+    extends TestKit(ActorSystem(clioDescription.replaceAll("\\s+", "-")))
+    with AsyncFlatSpecLike
+    with BeforeAndAfterAll
+    with Matchers
+    with RequestBuilding
+    with FailFastCirceSupport
+    with ModelAutoDerivation
+    with LazyLogging {
+
+  behavior of clioDescription
+
+  implicit val m: ActorMaterializer = ActorMaterializer()
+
+  /**
+    * Printer to override Circe's default marshalling behavior
+    * for None from 'None -> null' to omitting Nones entirely.
+    */
+  implicit val p: Printer = Printer.noSpaces.copy(dropNullKeys = true)
+
+  /**
+    * The web client to test against.
+    */
+  def clioWebClient: ClioWebClient
+
+  /**
+    * The URI of the Elasticsearch instance to test in a suite.
+    * Could point to a local Docker container, or to a deployed ES node.
+    */
+  def elasticsearchUri: Uri
+
+  /**
+    * HTTP client pointing to the elasticsearch node to test against.
+    *
+    * Needs to be lazy because otherwise it'll raise an
+    * UninitializedFieldError in the containerized spec because of
+    * the initialization-order weirdness around the container field.
+    */
+  lazy val elasticsearchClient: HttpClient = {
+    val host = HttpHost.create(elasticsearchUri.toString())
+    val restClient = RestClient.builder(host).build()
+    HttpClient.fromRestClient(restClient)
+  }
+
+  /**
+    * Get a bearer token for hitting the /api route of Clio.
+    *
+    * For now, just calls out to `gcloud` in an external process.
+    */
+  def bearerToken: String = "gcloud auth print-access-token".!!.stripLineEnd
```

Comment:
I believe as part of this checkin the tests no longer run inside a docker container.